### PR TITLE
[RHPAM-3640] Configure Kafka JBPM Emitter from Operator

### PR DESCRIPTION
**Pull request by desmax74**
7.11.0.CRX
Signed-off-by: desmax74 <mdessi@redhat.com>
See: https://issues.redhat.com/browse/RHPAM-3640

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[KIECLOUD-XYZ] Subject`, `[RHDM-XYZ] Subject` or `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by desmax74** [comment]
@spolti @jakubschwan

File: jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh [hunk 1]
```diff
@@ -776,6 +781,34 @@ function configure_kafka(){
...
+          fi
+          if [ -n "${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_CLIENT_ID}" ];then
+            JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.jbpm.event.emitters.kafka.client.id=${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_CLIENT_ID}"
+          fi
+          if [ -n "${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_MAX_BLOCK_MS}" ];then
+            JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.jbpm.event.emitters.kafka.max.block.ms=${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_MAX_BLOCK_MS}"
+          fi
+
+          JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.jbpm.event.emitters.kafka.topic.processes=jbpm-process-events"
+          JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.jbpm.event.emitters.kafka.topic.tasks=jbpm-task-events"
```
Comment (spolti) [inline jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh:802]: is this expected to be static value?

Comment (desmax74) [reply jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh:813]: Yep , in the docs https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.10/html-single/integrating_red_hat_process_automation_manager_with_other_products_and_components/index#kieserver-kafka-emit-proc_integrating-amq-streams and in the code : https://github.com/kiegroup/jbpm/blob/master/jbpm-event-emitters/jbpm-event-emitters-kafka/src/main/java/org/jbpm/event/emitters/kafka/KafkaEventEmitter.java#L163 https://github.com/kiegroup/jbpm/blob/master/jbpm-event-emitters/jbpm-event-emitters-kafka/src/main/java/org/jbpm/event/emitters/kafka/KafkaEventEmitter.java#L91..L113

Comment (jakubschwan) [reply jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh:813]: I would say, that these shouldn't be set to static value. This topic mapping does not make sense and force users to use only default topics. Also, here, we are setting defaults that are already in Kie Server - this is same as configuration done in Operator [see spolti's comment](https://github.com/kiegroup/kie-cloud-operator/pull/647#discussion_r625797943). 

In jbpm code is used method getProperty with default, so jbpm code first try to get mapping for process topic from system property `org.kie.jbpm.event.emitters.kafka.topic.processes` and if this system property is not set, then it is used default value, which is `jbpm-process-events`.

I think we can make have in Kie Server image and in Operator configuration points for jbpm event emitters kafka topic mapping.

**Comment by desmax74** [comment]
replaced by https://github.com/jboss-container-images/jboss-kie-modules/pull/456

**Pull request closed without merging**